wolfcrypt/src/aes.c

```diff
@@ -6146,12 +6233,12 @@ void AES_GCM_encrypt_avx2(const unsigned char *in, unsigned char *out,
 #endif /* HAVE_INTEL_AVX1 */
 
 #ifdef HAVE_AES_DECRYPT
-void AES_GCM_decrypt(const unsigned char *in, unsigned char *out,
+void AES_GCM_decrypt_aesni(const unsigned char *in, unsigned char *out,
```

Comment:
Earlier versions of AES functions with appending aesni use all caps `AES_ECB_decrypt_AESNI`, is there a reason to not have it be uniform for both CBC/ECB and GCM?

Reply:
`scripts/aes/x86/gcm_aesni.rb` and `scripts/aes/x86_64/gcm_aesni.rb` already used "_aesni", and I stayed consistent with that.  The tweaks in `scripts/aes/x86_64/xts_aesni.rb`  are with "_aesni" for consistency with them.  OTOH, I used "_AESNI" in `wolfssl/wolfcrypt/src/aes_asm.S` for consistency with bulk of existing code in `wolfssl/wolfcrypt/src/aes.c`.  it should be harmonized at some point, which will be easy because none of these implementations are public, but I wanted to punt for now on the decision of whether to go with "_aesni" or "_AESNI".  my preference is "_AESNI", FWIW.